bin/test-bpl.sh

```diff
@@ -0,0 +1,47 @@
+GOOD=""
```

Comment:
This file is a script to run the bpl translation verification (for that other repo).

I didn't do any commenting because we decided to do that functionality (bpl translation + validation, but no verification) as a separate pull request.

I left this in there because it is fully functional, and didn't know where else to store it until we do the second pull request.  The other things regarding bpl translation+validation were hacky and incomplete, so I just removed them (from `smack.py`, for example).

In any event, it'll need comments in the long run, so I'll comment it and commit/push again.